apps/staking/src/components/node/NodeHireNodeSection.tsx

```diff
@@ -38,7 +38,7 @@ export interface NodeHireNodeSectionProps {
 export const NodeHireNodeSection: FC<NodeHireNodeSectionProps> = (props) => {
     const { isHiring = false, onHire } = props;
     const { account } = useWallet();
-    const bg = useColorModeValue('white', 'gray.800');
+    const bg = useColorModeValue('white', 'rgba(255, 255, 255, 0.06)');
```

Comment:
If possible, please add a color var for `rgba(255, 255, 255, 0.06)` and reference it from the colors config instead.